terrarun/server/__init__.py

```diff
@@ -2634,7 +2634,14 @@ def _post(self, current_user, current_job, workspace_id):
 
         state_base64 = data.get("attributes", {}).get("state", None)
         if not state_base64:
-            return {}, 400
+            return {
```

Comment:
I'm avoiding approving this because I'm not sure how the ApiError is handled in the response - does it work?

Reply:
Most other cases, at the Api Resource level, it would need to return ApiError().get_api_details to produce valid JSON - what else would be handle to serialise this error to JSON :thinking:

Reply:
Oh right. I'm used to frameworks doing it for me

Reply:
I'll add the serialization and test it.

Reply:
```
{
	"errors": [
		{
			"status": "400",
			"title": "param is missing or the value is empty: state",
			"detail": "Terrarun does not support state upload directly to storage.",
			"source": {
				"pointer": "/data/attributes/state"
			}
		}
	]
}
```

Ok got the response I wanted.